docs/source/modules/cmci_action.rst

```diff
@@ -374,16 +374,14 @@ Examples
    - name: install a bundle in a CICS region
      cmci_action:
        cmci_host: "example.com"
-       cmci_port: "12345"
-       context: "plex_or_applid"
-       type: CICSBundle
-       action_name: install
-       resource:
+       cmci_port: 1234
+       context: "ABCDEFGH" # context is the name of your CICSplex in a CPSM environment or the applid of your region in an SMSS environment
+       scope: "CSYDEF/CSYSGRP" # scope only applies if you're in a CPSM environment and is either the name of a CICS system definition (CSYSDEF) or CICS system group (CSYSGRP)
```

Comment:
Again this needs to be an 8 char name.  Maybe use `IJKLMNOP` to follow the precedent you've already set?

Reply:
Thinking of mentioning 8 characters long somewhere in the comment. What do you say